storage/bbolt.go

```diff
@@ -101,11 +101,24 @@ func (b bboltStore) doTX(fn func(tx *bbolt.Tx) error, writable bool) error {
 		if err != nil {
```

Comment:
this error should also trigger a afterRollback

Reply:
except when the error is `ErrTxClosed` or `ErrTxNotWritable`

Reply:
why?

Reply:
hmmm yes, it should always do a onRollback. The caller is expecting the TX to be committed, if this doesn't happen it should be notified.

Reply:
fixed in lib